python/ray/tests/test_autoscaler.py

```diff
@@ -1889,6 +1933,62 @@ def testRecoverUnhealthyWorkers(self):
                 "ray-legacy-worker-node-type (lost contact with raylet)." in
                 events), events
 
+    def testTerminateUnhealthyWorkers(self):
+        """Test termination of unhealthy workers, when
+        autoscaler.disable_node_updaters == True.
+
+        Modified copy-paste of testRecoverUnhealthyWorkers.
+        """
+        config_path = self.write_config(SMALL_CLUSTER)
```

Comment:
Same here, would it be possible to parametrize `testRecoverUnhealthyWorkers`?

Reply:
Would rather not, since the functionality tested is different and I had to think for more than 30 sec on how to refactor to unify the two tests.